test/cypress/integration/group-paying.spec.js

```diff
@@ -183,9 +183,9 @@ describe('Group Payments', () => {
     cy.getByDT('link-PaymentRowSent').click()
     cy.getByDT('payList').find('tbody').children().should('have.length', 2)
     cy.getByDT('payList').within(() => {
-      cy.getByDT('payRow').eq(1).find('td:nth-child(1)').should('contain', `user3-${userId}`)
-      cy.getByDT('payRow').eq(1).find('td:nth-child(2)').should('contain', '$100')
-      cy.getByDT('payRow').eq(1).find('td:nth-child(3)').should('contain', humanDateToday)
+      cy.getByDT('payRow').eq(0).find('td:nth-child(1)').should('contain', `user3-${userId}`)
+      cy.getByDT('payRow').eq(0).find('td:nth-child(2)').should('contain', '$100')
+      cy.getByDT('payRow').eq(0).find('td:nth-child(3)').should('contain', humanDateToday)
```

Comment:
It's unclear to me why this was changed. Could you elaborate, was this intentional? If so, why?

Reply:
Yes, it was intentional. The new distribution function appeared to give the results in a different order, breaking this cypress test in the process.

The solution was to correct the row-index which cypress checked for payments by this user.

IE: cypress had correct values, but was checking in the wrong row for them.

Maybe we should rewrite this so that the of the payments listed doesn't matter, or maybe sort the payment distributions by amount.

Reply:
How is this possible? These tests are passing on `master`...

Reply:
Damn. Good. Point.

My initial thoughts are that perhaps the adjust-last algorithm somehow swapped the two payments value, but the name of the "to-user" of the payments are also swapped.

Definitely a puzzle.

Looking into.